### PR TITLE
/send function

### PR DESCRIPTION
**Pull request by landryarki**
/send uuid message
the client wan now use "" to make a sentence as an unique argument

### CODE REVIEW

File: src/client/get_response.c [hunk 1]
```diff
@@ -19,5 +19,7 @@ void get_response(client_struct_t *client)
     client->response = receive_package(client->socket);
     if (client->response == NULL)
         return;
-    client->responses_args = split_str(client->response, '|');
+    client->responses_args = str_to_word_array(client->response, "|");
+    for (int i = 0; client->responses_args[i] != NULL; i++)
```
Comment (Peralban) [inline src/client/get_response.c:23]: The print it's debug no ?

**Review by Peralban — APPROVED** [review]
Just one question but it's fine

File: src/client/get_command.c
Comment (Quentintnrl) [inline src/client/get_command.c:1]: A quoi servent ces changements ?

**Review by Quentintnrl — APPROVED** [review]

File: src/client/get_response.c (hunk 1 above)
Comment (landryarki) [reply src/client/get_response.c:23]: yeah, forgot to remove it, but well it can stay for the moment i guess

**Pull request merged**